### ISSUE TITLE
`s-function.json` seem to be weirdly constructed

### ISSUE THREAD

**Issue by Nopik** [serverless/serverless#361]
So, when I did `sls module create` with `hello` and `world` as names, I've got this `s-function.json`:

```
{
  "functions": {
    "HelloWorld": {
      "custom": {
        "excludePatterns": [],
        "envVars": []
      },
      "handler": "modules/hello/world/handler.handler",
      "timeout": 6,
      "memorySize": 1024,
      "endpoints": {
        "hello/world": {
          "method": "GET",
          "authorizationType": "none",
          "apiKeyRequired": false,
          "requestParameters": {},
          "requestTemplates": {
            "application/json": ""
          },
          "responses": {
            "400": {
              "statusCode": "400"
            },
            "default": {
              "statusCode": "200",
              "responseParameters": {},
              "responseModels": {},
              "responseTemplates": {
                "application/json": ""
              }
            }
          }
        }
      },
      "pathFunction": "/Users/nopik/Documents/Projects/slspc/back/modules/hello/world/s-function.json",
      "name": "world"
    }
  }
}
```

I see following problems with it:

1) Why `HelloWorld` function has `name: "world"` inside? Isnt the whole `name` property unnecessary?
2) `pathFunction` is an absolute path? So, if I commit it and another dev in my team will pull, it wont work?
3) `pathFunction` seems to be a path to the file itself. Is it really necessary to store that path there?

**Comment by visualasparagus** [comment]
Also, shouldn't the runtime field of the `s-module.json` be part of the `s-function.json`?

**Comment by ac360** [comment]
@Nopik The `name` and `pathFunction` were not supposed to be in there.  This is fixed in v0.0.13

@visualasparagus We'd like to have `runtime` at the module level.  It helps us greatly, since we will be mixing programming languages in this framework to support all Lambdas.  Modules should follow single runtimes.
